addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -2585,14 +2600,27 @@ const RangeUserValueWidget = UnitUserValueWidget.extend({
         this._value = ev.target.value;
         this._onUserValueChange(ev);
     },
+    /**
+     * @private
+     * @param {string} inputValue 
+     */
+    _computeDisplayValue(inputValue) {
+        if (this.toRatio) {
```

Comment:
Maybe remove `this.toRatio` since we no longer need to assign it to a variable.
```suggestion
        if (this.el.dataset.toRatio) {
```

Reply:
@bvr-odoo Indeed, you're right